src/js/modules/popupRegistration.js

```diff
@@ -54,13 +51,22 @@ function gratitudeOpen() {
 	registerModalGratitude.classList.add("open");
 }
 
-function popupClose(popupContent) {
+function modalClose() {
```

Comment:
why do you need `modalClose` function if you use its code only in `popupsClose` and nowhere else?
move its code to popup close and remove `closeModal`

Reply:
Done